### PR TITLE
feat(intake-formats): Migrate umbrella-dns to ingest

### PR DESCRIPTION
**Pull request by yosradirgham**
Nothing in the change-log as I except the behavior to be ISO with Logstash.

Parent Task: [#41572](https://github.com/SekoiaLab/platform/issues/41572)

### CODE REVIEW

File: Umbrella/umbrella-dns/ingest/parser.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      name: dsv.parse-dsv
+      properties:
+        input_field: original.message
+        output_field: message
+        columnnames:
+          - datetime
+          - user_name
+          - DATA
+          - source_ip
+          - destination_ip
```
Comment (rombernier) [inline Umbrella/umbrella-dns/ingest/parser.yml:14]:
```suggestion
          - source_nat_ip
```

File: Umbrella/umbrella-dns/ingest/parser.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          action.name: "DNS query"
+          action.target: "network-traffic"
+          action.type: "{{parsed_event.message.action_type}}"
+          destination.ip: "{{parsed_event.message.destination_ip}}"
```
Comment (rombernier) [inline Umbrella/umbrella-dns/ingest/parser.yml:31]:
```suggestion
          source.nat.ip: "{{parsed_event.message.source_nat_ip}}"
```

**Review by rombernier — COMMENTED** [review]
We would like to rename the field destination.ip by source.nat.ip as requested in the issue : https://github.com/SekoiaLab/platform/issues/44777

File: Umbrella/umbrella-dns/ingest/parser.yml (hunk 2 above)
Comment (rombernier) [reply Umbrella/umbrella-dns/ingest/parser.yml:31]: link to issue : https://github.com/SekoiaLab/platform/issues/44777

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**